runconfig/hostconfig_windows.go

```diff
@@ -10,25 +10,22 @@ import (
 // DefaultDaemonNetworkMode returns the default network stack the daemon should
 // use.
 func DefaultDaemonNetworkMode() container.NetworkMode {
-	return container.NetworkMode("default")
+	return container.NetworkMode("NAT")
```

Comment:
The network name are case sensitive so --net=NAT is not same as --net=nat. We will now support only --net=nat